Documentation/install/upgrade.rst

```diff
@@ -7,12 +7,165 @@ Upgrade Guide
 Kubernetes Cilium Upgrade
 =========================
 
-Cilium should be upgraded using Kubernetes rolling upgrade functionality in order to minimize network disruptions for running workloads.
+Cilium should be upgraded using Kubernetes rolling upgrade functionality in
+order to minimize network disruptions for running workloads.
+
+If you have followed the installation guide from :ref:`ds_deploy`, you probably
+have deployed a single ``cilium.yaml`` file. That file contains all the
+necessary components to run Cilium in your Kubernetes cluster. Those components
+were a `ConfigMap`, a ``ServiceAccount``, a ``DaemonSet``, and the ``RBAC`` for
```

Comment:
Nit (not very important, you can merge without this) \`ConfigMap\` --> \`\`ConfigMap\`\`

Reply:
We have a URL for the ConfigMap. It is intended to be `ConfigMap` and not ``ConfigMap``